### PR TITLE
Adds tests for new session logic

### PR DESCRIPTION
**Pull request by arunava59**

### CODE REVIEW

File: app/jobs/stale_session_terminator_job.rb [hunk 1]
```diff
@@ -3,8 +3,9 @@ class StaleSessionTerminatorJob < ApplicationJob
   sidekiq_options queue: 'default'
 
   def perform
-    Session.where(status: 'in-progress').where("? - last_ping_time >= INTERVAL '10 minutes'", Time.now.utc).in_batches do |batch|
-      batch.update_all("end_time=last_ping_time + (interval '10 minutes'),status=#{Session.statuses['done']}")
+    Session.where(status: 'in-progress')
+           .where("? - last_ping_time >= INTERVAL '#{Session::PING_INTERVAL_IN_MINUTES} minutes'", Time.now.utc).in_batches do |batch|
```
Comment (nronas) [inline app/jobs/stale_session_terminator_job.rb:7]: ❤️

File: spec/requests/api/v1/sessions_spec.rb [hunk 2]
```diff
@@ -1,26 +1,38 @@
...
-    expect {
-      post(api_v1_sessions_path, headers: headers, params: params, as: :json)
-      in_progress_session.reload
-    }.to change(in_progress_session, :status).from('in-progress').to('done')
+RSpec.shared_examples 'no active session' do
+  context 'when it is home session' do
+    let(:params) { { session: { type: 'home' } } }
+
+    it 'should create new home session' do
+      post '/api/v1/sessions/ping', params: params, headers: headers, as: :json
```
Comment (nronas) [inline spec/requests/api/v1/sessions_spec.rb:8]: can we check that an actual session was created in the DB here?

File: spec/requests/api/v1/sessions_spec.rb [hunk 3]
```diff
@@ -1,26 +1,38 @@
...
+      expect(json_response[:session_type]).to eq('home')
+    end
+  end
+
+  context 'when it is away session' do
+    let(:params) { { session: { type: 'away' } } }
+
+    it 'should create new away session' do
+      post '/api/v1/sessions/ping', params: params, headers: headers, as: :json
+      json_response = response.parsed_body.with_indifferent_access
```
Comment (nronas) [inline spec/requests/api/v1/sessions_spec.rb:20]: same thing here

File: spec/requests/api/v1/sessions_spec.rb (hunk 2 above)
Comment (arunava59) [reply spec/requests/api/v1/sessions_spec.rb]: Updated! Please have a look!

File: spec/requests/api/v1/sessions_spec.rb (hunk 3 above)
Comment (arunava59) [reply spec/requests/api/v1/sessions_spec.rb:26]: Updated! Please have a look!

**Pull request merged**